dataquality/loggers/base_logger.py

```diff
@@ -368,7 +374,7 @@ def label_idx_map(self) -> Dict[str, int]:
         >>> label_idx_map(labels)
         {"O": 0, "B-PER": 1, "I-PER": 2, "B-LOC": 3, "I-LOC": 4}
         """
-        return {label: idx for idx, label in enumerate(self.logger_config.labels or [])}
+        return self._label_idx_map(tuple(self.logger_config.labels))
```

Comment:
if labels is none you'll have `(None)` passed in. You may want to make it 
```suggestion
        return self._label_idx_map(tuple(self.logger_config.labels or []))
```